src/main/java/de/uni_passau/fim/se2/pipeline_helper/checkers/line_length/FileLineLengthViolations.java

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: 2022 Pipeline Helper Contributors
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+package de.uni_passau.fim.se2.pipeline_helper.checkers.line_length;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public record FileLineLengthViolations(Path file, int count, Map<Integer, Integer> violationsWithLength) {
```

Comment:
A `Map` does not ensure the lines remain in ascending order. Maybe a https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/SortedMap.html would be better?

Reply:
For sure better. Adjusting right now.